SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -269,6 +270,50 @@ void USpatialPackageMapClient::ClearRemovedDynamicSubobjectObjectRefs(const Work
 	}
 }
 
+void USpatialPackageMapClient::RemoveBNetLoadOnClientRuntimeRemovedComponents(const Worker_EntityId& EntityId,
+																			  const TArray<SpatialGDK::ComponentData>& NewComponents,
+																			  const SpatialGDK::ActorSystem& ActorSystem
+
+)
+{
+	auto ContainedInComponentsArr = [this, &NewComponents, &EntityId](const FUnrealObjectRef CheckComponentObjRef) {
+		for (const SpatialGDK::ComponentData& Component : NewComponents)
+		{
+			uint32 Offset = 0;
+			NetDriver->ClassInfoManager->GetOffsetByComponentId(Component.GetComponentId(), Offset);
```

Comment:
we could save a bit of time here by doing a similar check to the actor system:
```
// Early out if this isn't a generated component.
	if (ComponentId < SpatialConstants::STARTING_GENERATED_COMPONENT_ID && ComponentId != SpatialConstants::DORMANT_COMPONENT_ID)
	{
		return;
	}
	```
just continuing in our case

Reply:
added